src/OrleansAWSUtils/AWSUtilsHostingExtensions.cs

```diff
@@ -29,6 +32,30 @@ public static class AwsUtilsHostingExtensions
         }
 
         /// <summary>
+        /// Configure ClientBuilder with DynamoDBGatewayProvider
+        /// </summary>
+        /// <param name="builder"></param>
+        /// <param name="configureOptions"></param>
+        /// <returns></returns>
+        public static IClientBuilder UseDynamoDBGatewayProvider(this IClientBuilder builder,
```

Comment:
I used GatewayProvider consistently through the code. I used that one because GatewayProvider is the word got mentioned a lot in our legacy ClientConfiguration. So I think user are more used to GatewayProvider instead of GatewayListProvider. But this is debatable. Any opinion @jdom ?

Reply:
no strong opinion. Agree with being consistent throughout the configuration. I don't care if it's GatewayListProvider or GatewayProvider, but whatever we choose needs to apply to both the extension methods and the Option classes.

Reply:
Offline discussion agreed on GatewayListProvider